main/src/main/java/org/qbicc/main/Main.java

```diff
@@ -495,6 +495,17 @@ void call0(BaseDiagnosticContext initialContext) {
                                 builder.addBuilderFactory(Phase.ADD, BuilderStage.OPTIMIZE, SimpleOptBasicBlockBuilder::new);
                                 builder.addBuilderFactory(Phase.ADD, BuilderStage.INTEGRITY, ReachabilityBlockBuilder::new);
                                 builder.addBuilderFactory(Phase.ADD, BuilderStage.INTEGRITY, StaticChecksBasicBlockBuilder::new);
+                                builder.addPostHook(Phase.ADD, ctxt -> {
```

Comment:
Not thrilled with having a big block of code here, but working around a circular dependency problem.  This code can't go in either reflection or reachability without causing a cycle through intrinsics, so if we don't want to put it in main, we need to add a new plugin just to have this that can import both reflection and reachability.